### PR TITLE
Updated doc info about thumbnail's size.

### PR DESCRIPTION
**Pull request by 23rd**
I noticed again that the documentation still says about the 90 pixel limit.
After six months of testing, I can say that the formula `width + height <= 400 pixels` works perfectly for thumbnails.
I don't see any reason to mislead users with wrong limitation.

### CODE REVIEW

**Comment by delivrance** [comment]
Hi, I know you tested a lot on this and I had no reason to edit this info, until the Bot API has updated, so I took the value straight from their docs.

I prefer to mention official and conservative information about limits, when available, instead of relying on amateur tests that push limits to their edges (this is what I call misleading actually, because we can't be 100% sure this will remain true forever; it is not written anywhere).

Also, saying you **should** doesn't mean you cannot try higher values by yourself and use them if you see they work (there's no hard limit in Pyrogram in that sense), it just implies the method guarantees that within said limits you are safe.

tl;dr: We can't really guarantee that the 400px sum limit is not going to be changed in future, better stick with official limits so that people can't blame the library in case something goes wrong.

**Comment by 23rd** [comment]
Oh, I thought officially limits are nowhere to be found except in the code of official applications.
Can you share a link?

**Comment by delivrance** [comment]
The update is relatively new indeed. Example: https://core.telegram.org/bots/api#senddocument
Also search for "90" throughout the page

**Comment by 23rd** [comment]
Okay, I see. 
Have no any questions.
Thanks.
